src/tile-board/Tile.css

```diff
@@ -1,12 +1,46 @@
 .tile {
-  width: 50px;
-  height: 50px;
+  width: 42px;
+  height: 42px;
+  margin: 1.5px;
+  transition-duration: .25s;
+  transition-property: transform, var(--bs-btn-hover-bg);
+  animation: tileStart .6s cubic-bezier(0.42, 0, 0.45, 1.57) backwards;
```

Comment:
I really like that `backwards` animation fill mode means you can later transition the transform on hover without needing an extra container/wrapper to apply a separate transition to. Sweet!